README.md

```diff
@@ -102,6 +104,11 @@ See [Security filters for trimming Azure Search results using Active Directory i
 * Open the DotNetHowToAutocomplete.sln project in Visual Studio
 * Compile and Run the project
 
+## Running the DotNetHowToEncryptionUsingCMK sample
+
+* Open the DotNetHowToEncryptionUsingCMK.sln project in Visual Studio

```

Comment:
I think you need to explain a lot more in terms of how to configure KeyVault and AAD to run this sample.